### ISSUE TITLE
SelectorFormat “BEM” is not recognized correctly

### ISSUE THREAD

**Issue by blmyr** [brigade/scss-lint#318]
scss-lint did not complain about this before updating to the version 0.30.0

``` bash
$ cat _island.scss
```

``` scss
.island {
  display: block;
  padding: 20px;
}

.island--bordered {
  border: 1px solid black;
}
```

``` bash
$ scss-lint _island.scss
_island.scss:7 [W] SelectorFormat: Selector `island--bordered` should be written in BEM (Block Element Modifier) format
$ scss-lint --version
scss-lint 0.30.0
```
